amqp/transport.py

```diff
@@ -54,7 +55,7 @@ class _AbstractTransport(object):
     """Common superclass for TCP and SSL transports"""
     connected = False
 
-    def __init__(self, host, connect_timeout):
+    def __init__(self, host, connect_timeout, write_timeout, read_timeout):
```

Comment:
@dallasmarlow actually, you're right. It is marked as private. However, it feels weird that we require it in the constructor signature but we check if it is None later on. I guess making it a keyword is not bad.